models/viewdata.py

```diff
@@ -0,0 +1,27 @@
+import logging
+
+import folium
+from folium.plugins import HeatMap
+
+from models.storage import Storage
+
+
+class ViewData:
+    def __init__(self, storage: Storage) -> None:
+        self.storage = storage
+
+    def create_map(self, x: float, y: float) -> None:
+        city_center = [x, y]
+        data_points = []
+        logging.info("Creating a map")
```

Comment:
usually in logs we add specific to action info so it can be tracked. Here you have just a message that something happens. If so, it is usually a debug level log